libass/ass.c

```diff
@@ -19,20 +19,12 @@
 #include "config.h"
 #include "ass_compat.h"
 
-#include <stdio.h>
-#include <stdlib.h>
```

Comment:
This file uses a lot of stuff in these system headers; just because it happens to include something else that happens to also include these doesn't mean we shouldn't include them here. I don't like this commit apart from the removal of includes that are completely unused in a particular file (like FT_OUTLINE_H in files that don't use anything from that header).

Reply:
`ass_utils.h` is essentially a sort of precompiled header with bunch of standard system includes that get included everywhere. So I think that it's not much sense to include them repeatedly. But I'm not holding that commit too dearly so if there is a consensus that excessive includes are necessary I'll drop it.